packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -327,6 +327,11 @@ class WebpackCLI {
         let options = this.compilerConfiguration;
         let outputOptions = this.outputConfiguration;
 
+        // enable/disable colors
+        if (typeof args.color !== 'undefined') {
+            coloretteOptions.enabled = Boolean(args.color);
+        }
```

Comment:
oh, stop, why we need it here? we already do it https://github.com/webpack/webpack-cli/pull/2042/files#diff-f7ca7db5e4f622982a83595e0f0bdef1a6704559c96f9b7602e1349fea350105R30

Reply:
Here its for stats.

Reply:
Please clarify

Reply:
Leftover from an initial commit when I implemented it only for `stats`.